packages/itwin/measure-tools/src/ui-2.0/MeasureToolsUiProvider.tsx

```diff
@@ -71,13 +67,13 @@ export class MeasureToolsUiItemsProvider implements UiItemsProvider {
         return [
           ToolbarItemUtilities.createGroupButton(
             "measure-tools-toolbar",
-            this._props.itemPriority!,
+            this._props?.itemPriority ?? 20,
             "icon-measure",
             MeasureTools.localization.getLocalizedString(
               "MeasureTools:MeasurementGroupButton.tooltip",
             ),
             ToolbarHelper.constructChildToolbarItems(tools),
-            { groupPriority: this._props.groupPriority },
+            { groupPriority: this._props?.groupPriority ?? 10 },
```

Comment:
@bsteinbk we're hoping for this provider to replace the measure tools provided by the standard content tools provider

Reply:
@achrysaetos @aruniverse - The separator line in the image above could only occur if the group priorty is being set on the other two buttons. Where is the definition of the other buttons in this toolbar coming from? Also what is the frontstage definition look like that is being used in this example, because it may be possible that the group priority is being set in fronstage appdata..  My only point is that most apps do not set a group priority so setting one by default may cause a separator to appear where one is not wanted.

Reply:
@aruniverse  @achrysaetos  Applogies...I am blaming it on old age.... I took time to actually look at me standard tool provider and see that it defaults the groupPriority to 10 just like you had it.  Sorry I should have looked before commenting initially.

Reply:
No problem, i was unaware of the group priority being a thing until now, good to know :)